src/main/java/com/cloudant/client/org/lightcouch/NoResourceException.java

```diff
@@ -1,30 +0,0 @@
-/**
```

Comment:
This was added in 851d7d9d82f46529e6013f407ca77559ecbfcd15 but it is not referenced anywhere. I think it may have been added to improve compatibility with sync-android, but since it is in a different package anyway I don't think it will serve that purpose.